postgres/coredb-pg-slim/Dockerfile

```diff
@@ -44,26 +55,31 @@ RUN apt-get update && apt-get install -y \
         # build deps for pg_partman
         build-essential \
         git \
-        postgresql-server-dev-15 \
+        postgresql-server-dev-${PG_MAJOR} \
         # pg_stat_statement's package
         postgresql-contrib \
         # postgresql server
-        postgresql-15 \
+        postgresql-${PG_MAJOR} \
         # extensions
-        postgresql-15-postgis-3 \
-        postgresql-15-cron \
-        postgresql-15-repack \
-        postgresql-15-pgaudit \
+        postgresql-${PG_MAJOR}-postgis-3 \
+        postgresql-${PG_MAJOR}-cron \
+        postgresql-${PG_MAJOR}-repack \
+        postgresql-${PG_MAJOR}-pgaudit \
         && rm -rf /var/lib/apt/lists/*
 
 COPY ./postgresql.conf /usr/share/postgresql/${PG_MAJOR}/postgresql.conf.sample
 
-# install pg_partman
-RUN git clone https://github.com/pgpartman/pg_partman.git && \
-        cd pg_partman && \
-        make install && \
-        cd ../ && \
-        rm -rf pg_partman
+  WORKDIR /git
+  RUN git clone --branch "v${PG_VECTOR_VER}" https://github.com/pgvector/pgvector.git
+  WORKDIR /git/pgvector
+  RUN set -xe; \
+        make; \
+        make install;
+  
+  WORKDIR / 
+  RUN rm -rf /git
+  
+  RUN trunk install --version ${PG_PARTMAN_VER} pg_partman && trunk install --version ${PGMQ_VER} pgmq
```

Comment:
I think we want to drop extensions from this image as the become available in trunk registry

Reply:
I kind of figured that, but didn't want to risk an issue.  Thanks!

Reply:
Sounds good! We can add them back if we see issues